apps/frontend/src/components/right-side/radioview/RadioView.tsx

```diff
@@ -0,0 +1,86 @@
+import {useState} from "react";
+import {FaPlayCircle, FaPauseCircle} from "react-icons/fa";
+import {GetConfiguration} from "../../../api";
+import {Base, Text, Column, Flex, Button, LayoutGridItem} from "../../../common";
+
+export const RadioView = () => {
+  const [audio, setAudio] = useState(new Audio(GetConfiguration<string>("radio.stream.url")));
+  const [isPlaying] = useState(false);
```

Comment:
- Unused state and unused events down here.
- You don't have to create handlers for pausing and starting the radio. Keep track of the status radio with `paused` property as mentioned in MDN docs: 
![image](https://user-images.githubusercontent.com/87608619/229110196-05fa8576-0043-4b03-afc7-3116fa98b747.png)
and for changing the styles you can do something like that:

```tsx
<Button  
    onClick={() => audio.play()}
    type="button"
    style={{
          backgroundColor: "#F1B941",
          color: audio.paused ? "black" : "blue",
          cursor: "pointer",
     }}
>
```

so as the same for the button which handles the pause event